app/controllers/api/subcollections/snapshots.rb

```diff
@@ -25,8 +23,7 @@ def snapshots_create_resource(parent, _type, _id, data)
       end
 
       def delete_resource_snapshots(parent, type, id, _data)
-        validation = parent.validate_remove_snapshot(id)
```

Comment:
yes, just link the PRs via comments.

It would be best, if merging this PR does not affect the other repos, but sometimes this can't be avoided.
If this is the case, make sure both PRs can be merged in a small time window. I.e. the depending PR should turn green once the other is merged and the depending should be reviewed and marked ok.